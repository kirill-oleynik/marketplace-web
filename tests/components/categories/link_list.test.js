const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const {
  CategoriesLinkList
} = require('../../../app/components/categories/link_list');

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <CategoriesLinkList
        t={(translation) => translation}
        categories={[
          { id: 1, title: 'Test1' },
          { id: 2, title: 'Test2' }
        ]}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
