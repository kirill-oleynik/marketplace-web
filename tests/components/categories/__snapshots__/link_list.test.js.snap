// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<div
  className="categories"
  style={Object {}}
>
  <h4
    className="categories-title"
  >
    categories.sidebarTitle
  </h4>
  <div>
    <CategoryLink
      category={
        Object {
          "id": 1,
          "title": "Test1",
        }
      }
    />
    <CategoryLink
      category={
        Object {
          "id": 2,
          "title": "Test2",
        }
      }
    />
  </div>
</div>
`;
